src/main/java/org/bricolages/streaming/filter/TextOp.java

```diff
@@ -65,4 +65,15 @@ String castStringForce(Object value) {
         if (!(value instanceof String)) return null;
         return (String)value;
     }
+
+    final Pattern nullCharsPattern = Pattern.compile("^\\x00+$");
+    String removeHeadNullChars(String str) {
```

Comment:
Why removing only heading characters instead of whole text?  It does not make sense.